libraries/ashrae/guideline36/4.12-fan-coil-unit.yml

```diff
@@ -0,0 +1,32 @@
+fan-coil-unit:
+  body: >
+    @prefix p: <urn:___param___#> .
+    @prefix brick: <https://brickschema.org/schema/Brick#> .
+    p:name a brick:Fan_Coil_Unit ;
+      brick:feeds p:zone ;
+      brick:hasPart p:fan, p:clg_coil, p:htg_coil, p:filter ;
+      brick:hasPoint 
+        p:fan_start_stop, p:fan_speed, p:fan_status, p:sa_temp, p:ra_temp, 
+        p:filter_pd, p:clg_signal, p:htg_signal, p:zone_temp .
+  optional: ['clg_coil', 'htg_coil', 'fan_status', 'ra_temp', 'filter_pd']
+  dependencies:
+    - template: https://brickschema.org/schema/Brick#HVAC_Zone
+      library: https://brickschema.org/schema/1.3/Brick
+      args: {"name": "zone"}
+    - template: fan
+      args: {"name": "fcu_fan"}
```

Comment:
The values of the `args` dictionary need to appear as parameters in the body. So, above, you'd want to change line 7 to have `p:fcu_fan`, or change the args here to read as `args: {"name": "fcu"}`

(I definitely need to go and create an LSP for these template files to avoid these kinds of issues...)

Reply:
Same deal with the `clg_coil`, `htg_coil` and `filter` params/dependencies here too